gemup.js

```diff
@@ -74,7 +75,7 @@
 
           // Pull out the image url and call done
           success: function(res) {
-            options.done($(res).find('Location').text(), geminiKey, bucket);
+            options.done($(res).find('Location').text(), key, bucket);
```

Comment:
This would be a breaking change: https://github.com/artsy/force/blob/ff6a11c86c21e456c73c7001c7610dcbbd6e3ddd/src/desktop/apps/consign/client/actions.js#L354-L357